GVFS/GVFS.Common/LibGit2RepoPool.cs

```diff
@@ -90,6 +125,20 @@ private void ReturnToPool(LibGit2Repo repo)
             }
         }
 
+        private void ResetTimer()

```

Comment:
suggestion: could make function name less generic to include which timer this is resetting in the method name. Someone just looking at this method in the `LibGit2RepoPool` class might not know which timer it was referring to.